go/enclave/db/storage.go

```diff
@@ -43,19 +43,13 @@ func NewStorage(backingDB ethdb.Database, chainConfig *params.ChainConfig, logge
 	}
 }
 
-func (s *storageImpl) StoreGenesisRollup(rol *core.Rollup) error {
+func (s *storageImpl) StoreGenesisRollupHash(rollupHash common.L2RootHash) error {
```

Comment:
This method only updates the genesis hash now, with storing the rollup performed by another method (`StoreRollup`).